tests/test_download_core.py

```
@@ -1222,6 +1221,291 @@ def test_cleanup_legacy_files(tmp_path):
+        )
+
+        # Prereleases call should include only prereleases
+        prerelease_releases_arg = prerelease_call[0][0]
+        assert len(prerelease_releases_arg) == 2  # v2.7.7-open.1 and v2.7.7-open.2
+        assert all(
+            r["tag_name"].endswith("-open.1") or r["tag_name"].endswith("-open.2")
+            for r in prerelease_releases_arg
+        )
+
+        # Verify result includes latest prerelease version
+        downloaded_apks, new_versions, failed, latest_version, latest_prerelease = (
```

Ruff (0.14.5)

Unpacked variable `downloaded_apks` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)